gravity.sh

```diff
@@ -140,6 +140,7 @@ gravity_swap_databases() {
     # Check if the backup directory exists
     if [ ! -d "${gravityBCKdir}" ]; then
       mkdir -p "${gravityBCKdir}"
+      chown pihole:pihole "${gravityBCKdir}"
```

Comment:
```suggestion
      mkdir -p "${gravityBCKdir}" && chown pihole:pihole "${gravityBCKdir}"
```

Adding in that `&&` so the `chown` is only called if the directory is created.

Reply:
You mean as a safeguard in case `mkdir` fails?

Reply:
Yes, and because it's impossible to `chown` a directory that doesn't exist. The `&&` is cleaner than doing a "If directory exists then chown directory".